src/python/bin/aqueduct

```diff
@@ -297,12 +331,25 @@ if __name__ == "__main__":
     if not os.path.isdir(base_directory):
         os.mkdir(base_directory)
     
-    if args.command == "server":
-        server(args.expose)
+    if args.command == "start":
+        try:
+            server_popen = server(args.expose)
+            ui_popen = ui(args.expose)
+            print(generate_welcome_message(args.expose))
+            server_popen.wait()
+            ui_popen.wait()
+        except (Exception, KeyboardInterrupt) as e:
+            print(e)
+            print('\nTerminating Aqueduct service...')
+            server_popen.kill()
+            ui_popen.kill()
+            print('Aqueduct service successfully terminated.')
+    elif args.command == "server":
+        print("Command [aqueduct server] has been deprecated in favor of [aqueduct start]")
     elif args.command == "install":
         install(args.system[0]) # argparse makes this an array so only pass in value [0].
     elif args.command == "ui":
-        ui(args.expose)
+        print("Command [aqueduct ui] has been deprecated in favor of [aqueduct start]")
```

Comment:
The deprecation message seems to be missing for `aqueduct server`.

English nit -- the square bracket look a little out of place. Can we just do: 
```aqueduct ui and aqueduct server have been deprecated; please use aqueduct start to run both the UI and backend servers```